src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/x64/softmax.cpp

```diff
@@ -0,0 +1,152 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "custom/single_layer_tests/classes/softmax.hpp"
+#include "utils/cpu_test_utils.hpp"
+
+using namespace CPUTestUtils;
+
+namespace ov {
+namespace test {
+namespace SoftMax {
+const auto notOptimizedCPUSpec = CPUSpecificParams{{}, {}, {"ref_any"}, "ref_any"};
```

Comment:
It looks like https://github.com/openvinotoolkit/openvino/pull/18094#discussion_r1252866486 has not been answered. @luweizhou2016 , we need to test proper implementations! Could you please help to clarify which exact implementation types are supported in the current version of oneDNN.

Reply:
@luweizhou2016, please help to answer the question, thanks.

Reply:
Just talked with @tiger100256-hu  about this. Considering upstreamONEDNN has only one jit unified implment for avx512 , avx2, and SSE.  Suggest to use empty filer for cpu implement type. So the first returned JIT implement type from onednn would be run. Also the forked case can be tested. Thx!